StringDecode.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Stack;
+
+// Time Complexity :O(len(outputString))
```

Comment:
Time and space complexities both are O(n).
https://leetcode.com/problems/decode-string/discuss/424563/Java-O(n)-solution-using-two-stacks-with-easy-explanation-and-comments